samples/test/translate_dataset_management_test.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {AutoMlClient} = require('@google-cloud/automl');
+const {Storage} = require('@google-cloud/storage');
+
+const cp = require('child_process');
+const uuid = require('uuid');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const CREATE_DATASET_REGION_TAG = 'translate_create_dataset';
+const IMPORT_DATASET_REGION_TAG = 'import_dataset';
+const DELETE_DATASET_REGION_TAG = 'delete_dataset';
+const LIST_DATASET_REGION_TAG = 'list_datasets';
+const GET_DATASET_REGION_TAG = 'get_dataset';
+const DATASET_ID = 'TRL8522556519449886720';
+const EXPORT_DATASET_REGION_TAG = 'export_dataset';
+
+describe('Automl Translate Dataset Tests', () => {
+  const client = new AutoMlClient();
+
+  it('should create, import, and delete a dataset', async () => {
```

Comment:
this should be split into three separate tests, all of which need to be able to operate independently. 
1. create a dataset - in the "it" portion it will create a dataset, and you'll have an "after" block that deletes the dataset.
2. import a dataset - if this needs a dataset to import, that creation will be done in a "before" block, the actually import in the "it" block, and the deletion of anything created in an "after" block
3. delete a dataset - create a dataset in a "before" block, delete it as part of the test in the "it" block

Reply:
also, relating to my overall comment it'll probably be split into three separate files